sdcm/nemesis.py

```diff
@@ -2799,7 +2799,9 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def disrupt_mgmt_restore(self):
-        def get_total_scylla_partition_size():
+        raise UnsupportedNemesis("'mgmt_restore' nemesis is too unstable.")
```

Comment:
can you add a reference to a SCT issue (open one if doesn't exist)

Reply:
@roydahan said in the Argus discussion following:
```
I'm not bothered by the mgmt_restore that fails and is a known issue
```
So, I would like to hear the answer for this question too. What is that `known issue`...

Reply:
AFAIK the "known issue" is https://github.com/scylladb/scylla-cluster-tests/issues/6842. 
But @dkropachev is working on resolving it. So the PR is not needed.